test/integrant/core_test.cljc

```diff
@@ -251,6 +251,15 @@
                        [:init [::b ::c ::e] 1]
                        [:init ::a [1]]])))))
 
+  (testing "with failing composite refs"
+    (reset! log [])
+    (is (thrown-with-msg?
+          #?(:clj clojure.lang.ExceptionInfo :cljs cljs.core.ExceptionInfo)
+          #"^Invalid composite key: \[:integrant.core-test/a :b\]. Every keyword must be namespaced.$"
+          (ig/init {[::a :b] :anything}))
+        (str "an informative exception should be provided when a composite-keyword"
```

Comment:
Let's not bother with the message on `is`. None of the other tests have it, so let's be consistent.